Standard.AI.Data.EntityIntelligence/Models/Data/TableColumnMetadata.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+namespace Standard.AI.Data.EntityIntelligence.Models.Data
```

Comment:
The convention is to use plural for namespaces,  this should be Models.Datas

Reply:
`Data` is used both in singular and plural form.

Reply:
I know, it is just a nit pick, I noticed the community pluralize things with an s even if it doesn’t make sense from language perspective

Reply:
I think it should be `Datas` too - it's not the English word - it's a naming convention like we says `Childs` and not `Children`